tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -435,7 +446,7 @@
         '/api/hosts/:id',
         '/api/hosts/:id',
         '/api/hosts/:id/enc',
-        '/api/hosts/:id/status',
+        '/api/hosts/:id/status/:type',
         '/api/hosts/:id/status/:type',
```

Comment:
I think it's necessary because goal of the test is tracking unintended API changes :)

Yes, `/api/hosts/:id/status/:type'` being there twice is expected since the endpoint is actually listed two times in API, each with the different method and description.